package.json

```diff
@@ -1,8 +1,27 @@
 {
-  "private": true,
+  "name": "etcd-website",
+  "version": "0.0.1",
+  "description": "etcd website.",
+  "main": "none.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/etcd-io/website.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/etcd-io/website/issues"
+  },
+  "homepage": "https://etcd.io",
+  "dependencies": {
+    "autoprefixer": "^10.2.5",
+    "postcss": "^8.2.8",
+    "postcss-cli": "^8.3.1"
```

Comment:
Oh, you're really up-to-date w.r.t. the postcss versions. Nice!

Reply:
Lot of trial and error there...